subsystems/motor/speedmodifiers/AccelerationCap.java

```diff
@@ -61,10 +62,11 @@ public AccelerationCap(PDP pdp) {
 	 */
 	@Override
 	public double modify(double inputSpeed) {
+		double voltage = DriverStation.getInstance().getBatteryVoltage(); // TODO add default for when data isn't available
```

Comment:
WPILib isn't transparent about what errors could be thrown -- we (I) should try to trigger errors and see what happens.